packages/server/modules/shared/index.ts

```diff
@@ -1,11 +1,12 @@
 import { db } from '@/db/knex'
 import { getStreamFactory } from '@/modules/core/repositories/streams'
+import { getWorkspaceRoleAndSeatFactory } from '@/modules/gatekeeper/repositories/workspaceSeat'
```

Comment:
if we need this in the shared module, pls move this repo function into the foss licensed `workspaceCore` module